packages/test/local-server-stress-tests/src/test/localServerStressOrderSequentially.spec.ts

```diff
@@ -0,0 +1,97 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { done, isOperationType, takeAsync } from "@fluid-private/stochastic-test-utils";
+
+import { makeGenerator, reducer, saveFailures, type StressOperations } from "../baseModel.js";
+import {
+	convertToRealHandles,
+	covertLocalServerStateToDdsState,
+	DDSModelOpGenerator,
+	loadAllHandles,
+	validateConsistencyOfAllDDS,
+	type DDSModelOp,
+	type OrderSequentially,
+} from "../ddsOperations.js";
+import {
+	createLocalServerStressSuite,
+	LocalServerStressModel,
+	type LocalServerStressState,
+} from "../localServerStressHarness.js";
+
+const orderSequentiallyReducer = async (
+	state: LocalServerStressState,
+	op: OrderSequentially,
```

Comment:
Suggest some different name here so that later on it's not mistakable for a single op
```suggestion
	orderSequentially: OrderSequentially,
```

Reply:
op is consistently used in the fuzz/stress frameworks, so i'm going to keep this one